versioned_docs/version-1.2/compatible_oci_registries.mdx

```diff
@@ -8,8 +8,8 @@ sidebar_position: 70
 This page contains a list of projects leveraging ORAS, as well as
 registries that are known to support [OCI Artifacts][artifacts].
 
-*Would like your registry and/or project listed here?
-Please [submit an issue](https://github.com/oras-project/oras-www/issues/new?assignees=&labels=adopter&template=submit_adopter.yml&title=%5BAdd+adopter%5D%3A).
+*Note that this list is not comprehensive yet. Would like your registry and/or project listed here?
```

Comment:
```suggestion
*Note that this list is not comprehensive. Would like your registry and/or project listed here?
```

I don't think it will ever be comprehensive